packages/ndla-ui/src/locale/messages-nn.js

```diff
@@ -660,6 +660,25 @@ const messages = {
       linkText: 'Gå til ndla.no for dagens innhald',
     },
   },
+  frontPageToolbox: {
+    heading: 'Verktøykassa',
+    text:
+      'Har du lyst til å bli god til å presentere, eller vil du lære å studere smartare ved hjelp av riktig studieteknikk? Treng du råd om korleis du les mest mogleg effektivt til eksamen? I verktøykassen til NDLA finn du masse gode tips og råd!',
+    linkText: 'Sjå alle tipsa i verktøykassen her',
+  },
+  frontpageMultidisciplinarySubject: {
+    heading: 'Tverrfagleg tema',
+    text:
+      'Dei tre tverrfaglege tema i læreplanverket tar utgangspunkt i aktuelle samfunnsutfordringar som krev engasjement og innsats frå einskildmenneske og fellesskapet i lokalsamfunnet, nasjonalt og globalt.',
```

Comment:
```suggestion
      'Dei tre tverrfaglege temaa i læreplanverket tar utgangspunkt i aktuelle samfunnsutfordringar som krev engasjement og innsats frå einskildmenneske og fellesskapet i lokalsamfunnet, nasjonalt og globalt.',
```
Og burde det egentlig vore "tek utgangspunkt"?

Reply:
ja

Reply:
Endring committa